apps/token-manager/app/src/screens/Holders.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import BN from 'bn.js'
 import { useConnectedAccount } from '@aragon/api-react'
 import {
+  addressesEqual,
```

Comment:
For style, let's move these utilities to be at the end of the import list (notice how elsewhere we place things like `textStyle()` and `useTheme()` at the end of the list).